app/components/date-range-filter.js

```diff
@@ -58,11 +58,11 @@ export default class SliderFilterComponent extends Component {
    */
   @action
   sliderChanged([min, max]) {
-    // because the slider returns unix epochs based on its own step increment,
-    // get the startOf() the min timestamp's year, and the endOf() of the max timestamp's year
-    const minStart = parseInt(moment(min, 'X').utc().startOf('year').format('X'), 10);
-    const maxEnd = parseInt(moment(max, 'X').utc().endOf('year').format('X'), 10);
+    // must round integers because range slider sometimes provides
+    // decimal values in the callback
```

Comment:
:(

Reply:
It may not matter now but I want to control the rounding so it's predictable. I don't know why nouislider is doing this.